os/gstructs/gstructs_field.go

```diff
@@ -121,3 +121,16 @@ func (f *Field) IsEmpty() bool {
 func (f *Field) IsNil(traceSource ...bool) bool {
 	return empty.IsNil(f.Value, traceSource...)
 }
+
+// PriorityName returns tag name first in StructTagPriority
+// Then returns Name if it doesn't have a tag name anything.
+func (f *Field) PriorityName() string {
```

Comment:
All the `tag` associated function names start with prefix `Tag`, and are defined in file `gstructs_tag.go`. Is it better renaming the function name to `TagPriorityName`, and moving it to file `gstructs_tag.go`?